src/components/Auth/components/Login.jsx

```diff
@@ -25,8 +27,11 @@ export const Login = () => {
             body: JSON.stringify({ email, password }),
         })
         const { token, record } = await res.json()
```

Comment:
I don't think the return is record and token. Might be data and token. Please check first.